test/functional/ui/wildmode_spec.lua

```diff
@@ -98,8 +98,8 @@ describe("'wildmenu'", function()
     ]]}
 
     -- cmdline CTRL-D display should also be preserved.
-    feed([[<C-\><C-N>]])
-    feed([[:sign <C-D>]])   -- Invoke cmdline CTRL-D.
+    feed([[<C-U>]])
+    feed([[sign <C-D>]])   -- Invoke cmdline CTRL-D.
     expect_stay_unchanged{grid=[[
```

Comment:
[still failing](https://travis-ci.org/neovim/neovim/builds/580949539), although now it's the one before this (line 92). It's simply a race, we should probably use `any=...` combined with `screen: expect_unchanged()`.

```
[  ERROR   ] test/functional/ui/wildmode_spec.lua @ 84: 'wildmenu' is preserved during :terminal activity
test/functional/ui/screen.lua:587: Row 1 did not match.
Expected:
  |*                         |
  |                         |
  |                         |
  |define  jump  list  >    |
  |:sign define^             |
Actual:
  |*0: !terminal_output!     |
  |                         |
  |                         |
  |define  jump  list  >    |
  |:sign define^             |
To print the expect() call that would assert the current screen state, use
screen:snapshot_util(). In case of non-deterministic failures, use
screen:redraw_debug() to show all intermediate screen states.  
stack traceback:
	test/functional/ui/screen.lua:587: in function '_wait'
	test/functional/ui/screen.lua:370: in function 'expect'
	test/functional/ui/wildmode_spec.lua:22: in function 'expect_stay_unchanged'
	test/functional/ui/wildmode_spec.lua:92: in function <test/functional/ui/wildmode_spec.lua:84>
```